packages/routing/src/index.ts

```
@@ -0,0 +1,163 @@
+} from './store'
+
+// Platform adapters
+export { WebAdapter, webAdapter } from './adapters/web'
+export { MobileAdapter, mobileAdapter } from './adapters/mobile'
+export { DesktopAdapter, desktopAdapter } from './adapters/desktop'
+
+// Utility functions
+export const createPlatformAdapter = (platform: Platform) => {
+  switch (platform) {
+    case 'web':
+      const { webAdapter } = require('./adapters/web')
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)